docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,165 @@
+# Using the NMDC API Graphical User Interface (GUI)
+
+## Retrieving and Submitting Metadata using the ___Find___ and ___Metadata___ API Endpoints
+
+Metadata describing NMDC data (e.g. studies, biosamples, data objects, etc.) may be retrieved or submittied with GET and POST requests, respectively, using the **[NMDC API Graphical User Interface (GUI)](https://api.microbiomedata.org/docs#/)** The API GUI provides an user interface for programmatic access to the NMDC data portal without needing to use the Command Line.
+
+Requests can include various parameters to filter, sort, and organize the requested information. Attribute names in the parameters will vary depending on the collection. The required syntax of the parameters will also vary, depending on if it is a ___find___ or a ___metadata___ endpoint. ___Find___ endpoints are designed to use more compact syntax (for example, filtering biosamples for an "Ecosystem Category" of "Plants" would look like `ecosystem_category:Plants` using the `GET /biosamples` endpoint). While ___metadata___ endpoints use [MongoDB-like language querying](https://www.mongodb.com/docs/manual/tutorial/query-documents/) (e.g. the same filter would look like `{"ecosystem_category": "Plants"}` using the `GET /nmdcshema/{collection_name}` endpoint with `collection_name` set to `biosample_set`.)
```

Comment:
optional: you can point to https://docs.openalex.org/how-to-use-the-api/get-lists-of-entities/filter-entity-lists as prior art for the compact syntax.

Reply:
Added it! [9eb3bc4](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/9eb3bc4afcb79d30fdd78738a054700b0beea4eb) Thank you!